source/developer/slash-commands.rst

```diff
@@ -152,14 +156,14 @@ Tips and Best Practices
 
 2. Posts size is limited to 16393 characters for servers running `Mattermost Server v5.0 or later <https://docs.mattermost.com/administration/important-upgrade-notes.html>`__. Use the `extra_responses <https://developers.mattermost.com/integrate/slash-commands/#parameters>`__ field to reply to a triggered slash command with more than one post.
 
-3. You can restrict who can create slash commands in `System Console > Integrations > Integration Management <https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins>`__.
+3. You can restrict who can create slash commands in `**System Console > Integrations > Integration Management** <https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins>`__.
```

Comment:
Bold formatting doesn't work.

Reply:
Ah, I had a feeling that would happen. :(